nitime/analysis/tests/test_granger.py

```diff
@@ -109,5 +109,10 @@ def test_GrangerAnalyzer():
     # Test inputting ij:
     g2 = gc.GrangerAnalyzer(ts1, ij=[(0, 1), (1, 0)])
 
+    # g1 agrees with g2
+    npt.assert_almost_equal(g1.causality_xy[0, 1], g2.causality_xy[0, 1])
```

Comment:
I think the root of the error we saw at b576cedc5d3872a77f90ef72f98f43653874bde8 is just that g1.causality_{xy,yx} does not have proper [1, 0] array, due to its implicit ij's lower-triangularity (see <https://github.com/nipy/nitime/blob/master/nitime/analysis/granger.py#L107-L112>).
On the other hand, g2.causality_* does because its explicit argument `ij` contains (1, 0).
Thanks.